### PR TITLE
Asherons Departure (80+) Bur Flagging

### PR DESCRIPTION
**Pull request by RelycX**
2006-12-ColdTracks

This update doesn't contain anything on Bur itself, only the quest for 80+ to get flagged to enter the portal to Bur.

Very limited PCAP was available for the Eastern, Northern, or Western Vault dungeons. Monster placement may need adjustments if more data becomes available.

### CODE REVIEW

**Comment by ziang4891** [comment]
Why are you including changes made by other people in their PRs.  This update needs to be cleaned up to only include your own changes against master.  Including other peoples PR changes has the potential to cause collisions or unintended reversions.

File: Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34460 Asheron's Departure Journal.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+VALUES (34460, 'ace34460-asheronsdeparturejournal', 8, '2020-06-08 03:17:36') /* Book */;
+
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (34460,   1,       8192) /* ItemType - Writable */
+     , (34460,   5,        300) /* EncumbranceVal */
+     , (34460,  16,          8) /* ItemUseable - Contained */
+     , (34460,  19,          0) /* Value */
+     , (34460,  22,       1000) /* AvailableCharacter */
+     , (34460,  33,          1) /* Bonded - Bonded */
+     , (34460,  53,        101) /* PlacementPosition - Resting */
```
Comment (ziang4891) [inline Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34460 Asheron's Departure Journal.sql]: PlacementPosition - Resting is a default that should not be included in the SQL files.

Comment (RelycX) [reply Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34460 Asheron's Departure Journal.sql]: > Why are you including changes made by other people in their PRs? This update needs to be cleaned up to only include your own changes against master. Including other peoples PR changes has the potential to cause collisions or unintended reversions.

This was totally my bad.  I'm still learning my way around github and I must've copied the wrong branch or something for my own commits.  Any way to fix this or do I need to start over?

File: Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34460 Asheron's Departure Journal.sql [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+     , (34460, 114,          1) /* Attuned - Attuned */
+     , (34460, 174,          6) /* AppraisalPages */
+     , (34460, 175,          6) /* AppraisalMaxPages */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (34460,  11, True ) /* IgnoreCollisions */
+     , (34460,  13, True ) /* Ethereal */
+     , (34460,  14, True ) /* GravityStatus */
+     , (34460,  19, True ) /* Attackable */;
+
```
Comment (ziang4891) [inline Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34460 Asheron's Departure Journal.sql]: The first three bools are covered by the PhysicsState Int property and Attackable True is a defualt, so none of these need to be defined.  Same applies to all weenies.

Comment (RelycX) [reply Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34460 Asheron's Departure Journal.sql]: Good to know!  Will try to remember that going forward.

File: Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34500 Western Vault Notes.sql [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+VALUES (34500,   1,   33554773) /* Setup */
+     , (34500,   3,  536870932) /* SoundTable */
+     , (34500,   8,  100668176) /* Icon */
+     , (34500,  22,  872415275) /* PhysicsEffectTable */;
+
+INSERT INTO `weenie_properties_book` (`object_Id`, `max_Num_Pages`, `max_Num_Chars_Per_Page`)
+VALUES (34500, 1, 1000);
+
+INSERT INTO `weenie_properties_book_page_data` (`object_Id`, `page_Id`, `author_Id`, `author_Name`, `author_Account`, `ignore_Author`, `page_Text`)
+VALUES (34500, 0, 4294967295, 'Ardry the Dubious', '', False, 'I have located a third repository of Realaidain treasures, in the mountains south of Stonehold at 58.0 N, 20.8 W. This one was already well-invested with Falatacot minions. I even came face to face with a horrifying Falatacot creature named Vizidh. Before the waves of Sclavus forced me to retreat, I did spy a storeroom that seemed to be full of ornate golden goblets, with a Golem already decommissioned by Falatacot attacks. The Falatacot clearly don''t want anyone to be able to follow Asheron''s trail. I suppose if there is foul play at work in Asheron''s long disappearance, we must consider the Falatacot to be the most likely culprits.');
```
Comment (ziang4891) [inline Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34500 Western Vault Notes.sql]: Missing ending new-line, which can cause issues with building the initial test loads, using scripts

File: Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34460 Asheron's Departure Journal.sql (hunk 1 above)
Comment (ziang4891) [reply Database/Patches/2006-12-ColdTracks/9 WeenieDefaults/Book/Writable/34460 Asheron's Departure Journal.sql]: Usually if I get too far down a rabbit hole, its best to just start fresh, as its can be really easy to make it worse.

**Comment by RelycX** [comment]
I goofed and grabbed a bunch of non-related commits in one of my branches.  I'm gonna close this, make the fixes that Ziang has suggested so far.

**Pull request closed without merging**